index.rst

```diff
@@ -54,7 +54,7 @@
 
 
 The LSST Stack Release Process
```

Comment:
I think wee should rename it, and avoid the world "Stack".
I would suggest to use "Science Pipelines" instead.

Reply:
Agreed